app/helpers/downloads_helper.rb

```diff
@@ -82,4 +85,20 @@ def download_text
       }
     }
   end
+
+  def switch_citation_text
+    # Should reflect the month/year of the last monthly release
+    current_month_year = Date.parse(Wdpa::S3.current_wdpa_identifier)
```

Comment:
I would perhaps be a little bit more cautious with the identifier from S3.
It's quite unlikely that goes down, but if that happens or if the identifier returned is not in the expected format, `Date.parse` will raise an exception. 

To avoid that I would suggest either putting that value into a separate variable and check it later, or add a begin-rescue block rescuing for ArgumentError to catch exceptions related to Date.parse and manage this possibility.

Reply:
Good point...I might add the current month/year as a fallback (here's hoping that the releases are regular and won't be interrupted or delayed)

Reply:
Uhm, I'm a bit undecided on that. Perhaps you could get any ProtectedArea object and get the month and year from the `created_at` date? `created_at` should reflect the release as those object are recreated each month.

Reply:
Actually before I change this, let me confirm with Heather whether this is advisable.

Reply:
Have been given permission to use current month and year.